### DISCUSSION TITLE
Health bar visually fills at start of map when it shouldn't (osu!taiko)

### DISCUSSION THREAD

**Discussion by 12beesinatrenchcoat** [ppy/osu#25043] [Q&A] [answered]
The health bar starts at 0 in osu!taiko, but at the start of the beatmap, it still animates to full (this seems to be visual; it immediately drops once a note is hit).

https://github.com/ppy/osu/assets/25379179/a119bb57-b97c-41ba-afcc-980bbd8885cc

(The health bar also animates when paused, but that's less important.)

https://github.com/ppy/osu/assets/25379179/fbad2588-e3f9-436e-8542-a9ddbb900aa9

**Comment by peppy** [comment, marked as answer]
Oops.
